oci/layout/oci_transport_test.go

```diff
@@ -176,12 +178,17 @@ func TestReferencePolicyConfigurationIdentity(t *testing.T) {
 	ref, tmpDir := refToTempOCI(t)
 	defer os.RemoveAll(tmpDir)
 
-	assert.Equal(t, tmpDir+":tagValue", ref.PolicyConfigurationIdentity())
+	fullyExplicitPath, err := explicitfilepath.ResolvePathToFullyExplicit(tmpDir)
```

Comment:
I do know that this is needed for macOS right now, but this is a bit weird to have in tests. This workaround gets copied all over the place…

At the very least, a comment explaining the `/private/var` situation would help make sure a next Linux-only maintainer does not break everything again.

Can we instead add a shared workaround, something like
```go
func init() {
// comment about what this is for
  os.Setenv("TMPDIR", filepath.EvalSymlinks(os.Getenv("TMPDIR")))
}
```
or something like that? Perhaps even a helper function, IIRC there are tests in multiple subpackages which need a similar workaround.

Reply:
This should have been resolved by #336.